views/index.html

```diff
@@ -18,27 +18,27 @@
           </div>
           <h3>Product Development</h3>
             <p>
-              We build <i>mobile-first</i> MVPs with intuitive and user-friendly interfaces.
+              We build responsive, mobile-first MVPs with intuitive and human-centred design.
             </p>
         </div>
       </div>
       <div class="pure-u-1 pure-u-md-1-3 flex-center section-box-wrapper">
         <div class="section-box">
           <div class="offering-icon user-testing-icon">
           </div>
-          <h3>User testing</h3>
+          <h3>Co-working</h3>
```

Comment:
Think SEO on this one. We just don't refer to our _community_ as users: https://github.com/dwyl/start-here/blob/master/manifesto.md#we-do-not-have-users-we-have-people